hedyorg/hedy
# 🪲 Escape internal var used in repeat

Fixes #5255 
The internal variable `i` used in repeat statements is now escaped with the `__` prefix, so that it does not show in the output window.

**How to test**
- Run Hedy locally and run the following snippets for the specified levels. Ensure that the `i` variable does not appear in the blue variables list in the upper right corner of the output window:

Level 7:
```
options is 1, 2, 3, 4, 5
repeat 5 times print options at random
```
Level 8:
```
options is 1, 2, 3, 4, 5
repeat 5 times
  print options at random
```
